plugins/polio/js/src/domains/Calendar/campaignCalendar/map/VaccinesLegend.tsx

```diff
@@ -1,15 +1,16 @@
-import React, { FunctionComponent } from 'react';
-import { Grid, Paper, Box, Typography } from '@mui/material';
+import { Box, Grid, Paper, Typography } from '@mui/material';
 import { useSafeIntl } from 'bluesquare-components';
+import React, { FunctionComponent } from 'react';
 import { useStyles } from '../Styles';
 
 import MESSAGES from '../../../../constants/messages';
-import { polioVaccines } from '../../../../constants/virus';
+import { polioVaccines, useMapLegend } from '../../../../constants/virus';
 
-type Vaccine = typeof polioVaccines[number];
+type Vaccine = (typeof polioVaccines)[number];
```

Comment:
Are you sure this is correct?
`typeof polioVaccines` will always be `"object"` so `(typeof polioVaccines)[number]` will be letter, no?